### PR TITLE
Updates policy action to include validation and publishing

### PR DESCRIPTION
**Pull request by scottford-io**
Signed-off-by: Scott Ford <scott@scottford.io>

### CODE REVIEW

File: policy/README.md [hunk 1]
```diff
@@ -26,22 +27,43 @@ Additionally, you need to specify the service account credentials as an environm
...
+  cnspec-policy-validate:
+    name: Validate cnspec policy bundle
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: mondoohq/actions/policy@scottford/updates-policy-action
+        env:
+          MONDOO_CONFIG_BASE64: ${{ secrets.MONDOO_SERVICE_ACCOUNT }}
+        with:
+          action: validate
```
Comment (chris-rock) [inline policy/README.md:49]: Isn't that conflicting with https://github.com/mondoohq/actions/tree/main/cnspec-lint? We also renamed it to lint

File: policy/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
-# Mondoo Policy Action
+# Mondoo policy bundle action
 
-A GitHub Action for publishing Mondoo policies to Mondoo Platform.
+A GitHub Action for publishing cnspec policy bundles to Mondoo Platform.
```
Comment (chris-rock) [inline policy/README.md:3]: We may want to refocus this one to publishing bundles

File: policy/README.md [hunk 3]
```diff
@@ -12,11 +12,12 @@ Adding policies to Mondoo Platform requires a [Mondoo service account](https://m
...
-| `service-account-credentials` | false    |         | Base64 encoded [service account credentials](https://mondoo.com/docs/platform/service_accounts/#creating-service-accounts) used to authenticate with Mondoo Platform. You can also use the environment variable mentioned below. |
+| Property                      | Required | Default  | Description                                                                                                                                                                                                                      |
+| ----------------------------- | -------- | -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `action`                      | false    | validate | Sets the action for cnspec bundle command: publish, validate (default "validate")                                                                                                                                                |
```
Comment (chris-rock) [inline policy/README.md:17]: I think we should focus this action on pure publishing

**Comment by scottford-io** [comment]
@chris-rock I am going to close this PR. I think we should have another conversation what we should do with this action.

**Pull request closed without merging**